channel-email/src/test/java/com/synopsys/integration/alert/channel/email/validator/EmailDistributionConfigurationValidatorTest.java

```diff
@@ -0,0 +1,165 @@
+package com.synopsys.integration.alert.channel.email.validator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+
+import com.synopsys.integration.alert.api.channel.CommonChannelDistributionValidator;
+import com.synopsys.integration.alert.api.common.model.errors.AlertFieldStatus;
+import com.synopsys.integration.alert.api.provider.ProviderDescriptor;
+import com.synopsys.integration.alert.channel.email.descriptor.EmailDescriptor;
+import com.synopsys.integration.alert.common.descriptor.ChannelDescriptor;
+import com.synopsys.integration.alert.common.descriptor.validator.ConfigurationFieldValidator;
+import com.synopsys.integration.alert.common.enumeration.FrequencyType;
+import com.synopsys.integration.alert.common.rest.model.FieldModel;
+import com.synopsys.integration.alert.common.rest.model.FieldValueModel;
+import com.synopsys.integration.alert.common.rest.model.JobFieldModel;
+
+class EmailDistributionConfigurationValidatorTest {
+    static final Set<AlertFieldStatus> REQUIRED_FIELDS_ERRORS = new HashSet<>(List.of(
+        AlertFieldStatus.error(ChannelDescriptor.KEY_CHANNEL_NAME, ConfigurationFieldValidator.REQUIRED_FIELD_MISSING_MESSAGE),
+        AlertFieldStatus.error(ChannelDescriptor.KEY_NAME, ConfigurationFieldValidator.REQUIRED_FIELD_MISSING_MESSAGE),
+        AlertFieldStatus.error(ChannelDescriptor.KEY_FREQUENCY, ConfigurationFieldValidator.REQUIRED_FIELD_MISSING_MESSAGE),
+        AlertFieldStatus.error(ChannelDescriptor.KEY_PROVIDER_TYPE, ConfigurationFieldValidator.REQUIRED_FIELD_MISSING_MESSAGE)
+    ));
```

Comment:
You should be able to use Set.of() rather than create a list and passing it into a hash set